docs/native/src/wopi_requirements.rst

```diff
@@ -53,7 +53,7 @@ Bootstrapper
 
 * :ref:`Bootstrap`
 * :ref:`GetNewAccessToken`
-* :ref:`Shortcut operations` (Of these, only :ref:`GetRootContainer (bootstrapper)` is required)
+* :ref:`GetRootContainer (bootstrapper)`, if :ref:`GetRootContainer (ecosystem)` isn't implemeneted
```

Comment:
This still doesn't seem accurate. Why not just list GetRootContainer (bootstrapper) as a firm requirement? I don't think you want partners _not_ implementing it, even though you can technically fall back to GetRootContainer (ecosystem).

Alternatively, add it to the 'not called currently but you still need to implement it' list below.